Objects/memoryobject.c

```diff
@@ -3200,4 +3289,4 @@ PyTypeObject PyMemoryView_Type = {
     0,                                        /* tp_init */
     0,                                        /* tp_alloc */
     memoryview,                               /* tp_new */
-};
+};
```

Comment:
Make sure the last line of the file ends in a newline character! (Most editors have a setting to make sure of this.)